dart-stage0.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: dart-stage0
+  version: 3.1.0
+  epoch: 0
+  description: "Dart is a client-optimized language for fast apps on any platform (temporary bootstrap package)"
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - zip
+
+pipeline:
+  - assertions:
+      required-steps: 1
+    pipeline:
+      - if: ${{build.arch}} == 'x86_64'
+        uses: fetch
+        with:
+          uri: https://storage.googleapis.com/dart-archive/channels/stable/release/${{package.version}}/sdk/dartsdk-linux-x64-release.zip
+          expected-sha256: 7ce5c1560b1d8ea5ee0e6d44f1c3e7b804ddc5377b38d72fe4d43009a6c67e84
+          extract: false
+      - if: ${{build.arch}} == 'aarch64'
+        uses: fetch
+        with:
+          uri: https://storage.googleapis.com/dart-archive/channels/stable/release/${{package.version}}/sdk/dartsdk-linux-arm64-release.zip
+          expected-sha256: fd4fcc05f1d1c82fd618b83c7d877968460c5bd425774d89add438be12a20795
+          extract: false
+
+  - if: ${{build.arch}} == 'x86_64'
```

Comment:
These steps should also be wrapped in a subpipeline with an assertion, like above.  That way the pipeline will fail if extract fails.